source/Ship.h

```diff
@@ -532,6 +536,9 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	double CalculateAttraction() const;
 	double CalculateDeterrence() const;
 
+	// Determine if this bay can hold a ship of this category.
+	bool BayContains(const Bay &bay, const std::string &category) const;
```

Comment:
this should be static imo

Reply:
I moved it to the namespace.

Reply:
this should be in Ship::Bay imo

Reply:
Either works for me

Reply:
That definitely seems like a logical place to put it.